xbmc/interfaces/legacy/ModuleXbmcgui.h

```diff
@@ -85,6 +86,30 @@ namespace XBMCAddon
 #endif
     ///@}
 
+#ifdef DOXYGEN_SHOULD_USE_THIS
+    ///
+    /// \ingroup python_xbmcgui
+    /// @brief \python_func{ xbmcgui.getMousePosition() }
+    ///-------------------------------------------------------------------------
+    /// Returns the current mouse position as a x,y integer tuple.
+    ///
+    /// @return                        The current mouse position
+    ///
+    ///
+    ///--------------------------------------------------------------------------
+    ///
+    /// **Example:**
+    /// ~~~~~~~~~~~~~{.py}
+    /// ..
+    /// pos = xbmcgui.getMousePosition()
```

Comment:
and unpack it here (`x, y = xbmcgui.getMousePosition()`) to make it obvious which is x and y